sdk/storage/azblob/internal/generated/zz_models.go

```diff
@@ -1720,7 +1720,7 @@ type UserDelegationKey struct {
 	SignedExpiry *time.Time `xml:"SignedExpiry"`
 
 	// REQUIRED; The Azure Active Directory object ID in GUID format.
-	SignedOid *string `xml:"SignedOid"`
+	SignedOid *string `xml:"SignedOID"`
```

Comment:
Was this, and the change on line 1732, intentional or due to find/replace.

Reply:
Ohh Goland must have caused this. Reverted it back